docs/en/getting-started/install.md

```diff
@@ -1,13 +1,34 @@
 ---
-sidebar_label: Installation
-sidebar_position: 1
-keywords: [clickhouse, install, installation, docs]
-description: ClickHouse can run on any Linux, FreeBSD, or Mac OS X with x86_64, AArch64, or PowerPC64LE CPU architecture.
-slug: /en/getting-started/install
-title: Installation
+sidebar_label: Install
+keywords: [clickhouse, install, getting started, quick start]
+slug: /en/install
 ---
 
-## System Requirements {#system-requirements}
+# Installing ClickHouse
+
+You have two options for getting up and running with ClickHouse:
+
+- **[ClickHouse Cloud](https://clickhouse.cloud/):** the official ClickHouse as a service, - built by, maintained, and supported by the creators of ClickHouse
+- **Self-managed ClickHouse:** ClickHouse can run on any Linux, FreeBSD, or Mac OS X with x86_64, AArch64, or PowerPC64LE CPU architecture
+
+## ClickHouse Cloud
+
+The quickest and easiest way to get up and running with ClickHouse is to create a new service in [ClickHouse Cloud](https://clickhouse.cloud/):
+
+<div class="eighty-percent">
+
+![Create a ClickHouse Cloud service](@site/docs/en/_snippets/images/createservice1.png)
```

Comment:
It's w/ `@site` in one place, w/o in other. And it used to be relative links, and now it's absolute. Not sure, why the absolute are better?

Reply:
`@site` gets used for images and include files.  They all convert to relative links.  We are having problems with `./foo.md` and `../../foo.md` links working on initial load of a page, but 404ing no reloads.  Using the `/docs/...` fixes this.  As I receive issues about 404's I am fixing the individual docs, and as I edit docs I am fixing them.